### ISSUE TITLE
Index.sort_values puts missing values at the start with ascending=False

### ISSUE THREAD

**Issue by TomAugspurger** [pandas-dev/pandas#31220]
#### Code Sample, a copy-pastable example if possible

Index.sort_values places missing values at the start of the result with `ascending=False`

```python
In [4]: pd.Index([1, np.nan, 0]).sort_values(ascending=False)
Out[4]: Float64Index([nan, 1.0, 0.0], dtype='float64')

In [5]: pd.Series([1, np.nan, 0]).sort_values(ascending=False)
Out[5]:
0    1.0
2    0.0
1    NaN
dtype: float64
```
#### Problem description

This differs from Series.sort_values, which always leaves NA values at the end.

#### Expected Output

```python
Out[4]: Float64Index([1.0, 0.0, nan], dtype='float64')
```

**Comment by dsaxton** [comment]
Should `Index.sort_values` also have an `na_position` argument? It looks like it just uses `Index.argsort` whereas `Series` handles the NA values separately according to `na_position`.

**Comment by Dr-Irv** [comment]
Agree with @dsaxton above

**Comment by junjunjunk** [comment]
take

**Comment by junjunjunk** [comment]
May I ask you a quick question?
I'll add na_position argument to Index.sort_values.
Then, which should I set the default value of na_position, first or last?

The default value of na_position in Series.sort_values is **last**.
On the other hand, the default value of na_position in Series.sort_values should be **first** in order to be non-destructive.
May I set the default value to "last" in order to make the behavior consistent with the Series along with this issue?
